PROJECT

```diff
@@ -19,6 +19,9 @@ resources:
   kind: VolumeReplicationGroup
   path: github.com/ramendr/ramen/api/v1alpha1
   version: v1alpha1
+  webhooks:
```

Comment:
The webhooks are no longer needed.  It was needed to do a `SubjectAccessReview` of the user specifying multiple namespaces.  Since we've switched to the admin namespace approach this information is available later and can be performed in the controller.  Additionally, there is the problem of VRG and Recipe being updated concurrently and the inability to make the validating webhook read and subsequent commit atomic.